models/comment.rb

```diff
@@ -24,6 +24,7 @@ class Comment < Content
 
   index({author_id: 1, course_id: 1})
   index({_type: 1, comment_thread_id: 1, author_id: 1, updated_at: 1})
+  index({comment_thread_id: 1, author_id: 1, created_at: 1})
```

Comment:
Almost all of our existing indices include "_type".  I wonder which of those are valid.  

Also, @tobz, which index did you think was unused that was actually used?  Did you learning anything from that that could help us avoid removing indexes that are actually used?

FYI: @BenjiLee (who did some index clean-up in the past)